### DISCUSSION TITLE
Build the latest CC-Tweaked for a previous minecraft version

### DISCUSSION THREAD

**Discussion by osvegn** [cc-tweaked/CC-Tweaked#1616] [Q&A] [answered]
I'm playing with CC-Tweaked in Minecraft 1.12.2, and I want to use methods from the inventory module.
Is it possible to build the latest CC-Tweaked source code for minecraft 1.12.2 ?

**Comment by SquidDev** [comment, marked as answer]
> Is it possible to build the latest CC-Tweaked source code for minecraft 1.12.2 ?

Assuming you mean "is it possible to retarget the latest CC:T code to use 1.12?" (rather than "does the latest 1.12 version still build?"). No - there's been a lot of changes, both to the mod and the base game in the last 5 years, and backporting those is pretty much impossible.

However, If you're on 1.12, you can install [Plethora](https://plethora.madefor.cc/) instead. It provides the same functionality as the inventory methods (though with some minor differences, see [its documentation for a list of methods](https://plethora.madefor.cc/methods.html#targeted-methods-net.minecraftforge.items.IItemHandler)).
